test/autorest/lrogroup/lrogroup_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package lrogrouptest
```

Comment:
Can more tests be added or is there not enough working yet?

Reply:
Hmm maybe add them once `Wait()` is implemented as that calls the other methods.

Reply:
Yes I was waiting to add all tests until we finalize most (if not all) aspects of the LROs.